.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: node_js
 node_js:
-  - "4.4.0"
+  - "node"
```

Comment:
No worries - had a bit of a time figuring out what the error `TypeError: Buffer.alloc is not a function` meant, finally figured out it just means nodejs testing was outdated.